### PR TITLE
ecc_schedule command

### PR DESCRIPTION
**Pull request by masokol**

### CODE REVIEW

**Comment by itskarlsson** [comment]
fixes #158

File: ecchronos-binary/src/bin/ecc_schedule.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+def print_repair_job(repair_job):
+    repair_jobs_table = [["Id", "Keyspace", "Table", "Status", "Repaired(%)",
+                          "Completed at", "Next repair", "Recurring"]]
+    repair_jobs_table.append(convert_repair_job(repair_job))
+    table_formatter.format_table(repair_jobs_table)
+
+
+def parse_arguments():
+    parser = ArgumentParser(description='Schedule a repair')
+    parser.add_argument('-k', '--keyspace', type=str,
```
Comment (itskarlsson) [inline ecchronos-binary/src/bin/ecc_schedule.py:49]: There are a few other options that might be interesting here.
For example, partitioner--range as noted in #158.
Would be interesting to add but would require a far more invasive change. Certainly an improvement for the future to think about.

**Review by itskarlsson — APPROVED** [review]
Add an entry in the changelog and make sure you format the commit message correctly. 
Otherwise, LGTM

File: ecchronos-binary/src/bin/ecc_schedule.py (hunk 1 above)
Comment (masokol) [reply ecchronos-binary/src/bin/ecc_schedule.py:49]: Agree, next up is merging the commands. After that i will see if i can fix all the issues mentioned in #158

**Pull request merged**